app/decorators/decidim/consultations/admin/response_form_decorator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Decidim::Consultations::Admin::ResponseForm.class_eval do
+  attribute :question_slug, String
+  attribute :blank_vote
+
+  validate :only_one_blank_vote_response
+
+  def only_one_blank_vote_response
+    question = Decidim::Consultations::Question.find_by(slug: question_slug) if question_slug
+    blank_vote_ = blank_vote == '1' ? true : false
+  
+    if question.responses.where(blank_vote: true).count == 1 && blank_vote_ && id != question.responses.find_by(blank_vote: true).id
```

Comment:
Also, if you want to make it a little more estable, add on this conditional if question is blank? or also do it like this:
`
if question&.responses&.where(blank_vote: true).count == 1 && blank_vote_ && id != question&.responses&.find_by(blank_vore: true)&.id
`